roles/pulp-webserver/templates/pulp-vhost.conf.j2

```diff
@@ -5,7 +5,7 @@ Define pulp-api {{ pulp_api_host }}:{{ pulp_api_port }}
   ServerName {{ ansible_nodename }}
 
   ## Vhost docroot
-  DocumentRoot {{ pulp_webserver_static_dir }}
+  DocumentRoot {{ pulp_user_home }}
```

Comment:
Are you sure we want to share out the entire dir? Although it will normally be a system user account (`useradd --system`), there could be some other user-account-related files under there.

Reply:
actually I'm not familiar with apache and nginx, this PR was pure trial and error, this was the only way I found to make this work. Could you help me to find a better way to address it?